test/pkg/containerd/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# alpine 3.5 lacks go1.8
+FROM golang:1.8
```

Comment:
If you use the `linuxkit/alpine` package you get golang 1.8. See for example: https://github.com/linuxkit/linuxkit/blob/master/pkg/sysctl/Dockerfile